react-design/my-app/src/RotatingBanner.jsx

```diff
@@ -1,19 +1,16 @@
 import Banner from './Banner';
 import NextBtn from './NextBtn';
 import PrevBtn from './PrevBtn';
-import Carousel from './Carousel';
+import Indicators from './Indicators';
 
-export default function RotatingBanner() {
+export default function RotatingBanner({ items }) {
+  // const [count, setCount] = useState(0);
   return (
-    <>
-      <Banner />
-      <div>
-        <PrevBtn />
-      </div>
-      <Carousel />
-      <div>
-        <NextBtn />
-      </div>
-    </>
+    <div>
+      <Banner item={items[0]} />
```

Comment:
Instead of passing `items[0]`, we should update our code to pass `items[currentIndex]` instead in Phase 2. 

Play around and  try passing first `items[0`] then pass `items[currentIndex]` as props and figure out why `items[0]` won't work for moving through the carousel (i.e would pressing the next or prev button work?)